src/test/java/nl/knaw/dans/pf/language/ddm/api/Ddm2EmdCrosswalkTest.java

```diff
@@ -285,6 +285,30 @@ public void languageWithSchemeAndId() throws Exception {
     }
 
     @Test
+    public void languageWithSchemeAndIdButNoEmdCode() throws Exception {
+        // @formatter:off
+        String ddm = "<?xml version='1.0' encoding='utf-8'?><ddm:DDM"
+            + "  xmlns:dc='http://purl.org/dc/elements/1.1/'"
+            + "  xmlns:ddm='http://easy.dans.knaw.nl/schemas/md/ddm/'"
+            + "  xmlns:dcterms='http://purl.org/dc/terms/'"
+            + "  xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'>"
+            + "  <ddm:dcmiMetadata>"
+            + "    <dc:language xsi:type='dcterms:ISO639-2'>gre</dc:language>"
+            + "  </ddm:dcmiMetadata>"
+            + "</ddm:DDM>";
+        // @formatter:on
+
+        DefaultElement top = firstEmdElementFrom(ddm);
+        assertThat(top.elements().size(), is(1));
+        assertThat(top.getQualifiedName(), is("emd:language"));
+
+        DefaultElement sub = (DefaultElement) top.elements().get(0);
+        assertThat(sub.getQualifiedName(), is("dc:language"));
+        assertThat(sub.getText(), is("gre"));
+        assertThat(sub.attributeCount(), is(0));
```

Comment:
So, if in the DDM the `<dc:language>` has `xsi:type='dcterms:ISO639-2'`, but it isn't one from the hardcoded list above, you don't get an `ISO 639` scheme in the resulting EMD anymore? Can you describe the effect of this on the webui? What would it look like there and what would be different for these 'languages without scheme'? Are they still displayed, editable, deletable, etc.?